swagger/v1/api_spec.json

```diff
@@ -2211,7 +2371,7 @@
             "description": "The event declaration date",
             "type": "string",
             "format": "date-time",
-            "example": "2021-05-31T02:21:32.000Z"
+            "example": "2021-05-31T03:21:32.000+01:00"
```

Comment:
This is wrong. Shouldn't be generating an offset to nowhen.

Reply:
Fixed